src/fsharp/FSharp.Core.Unittests/FSharp.Core/Microsoft.FSharp.Collections/StringModule.fs

```diff
@@ -41,8 +41,8 @@ type StringModule() =
         let e6 = String.concat "foo" ["bar"]
         Assert.AreEqual("bar", e6)
         
-        let e7 = String.concat "foo" ["bav";"baz"]
-        Assert.AreEqual("bavfoobaz", e7)
+        let e7 = String.concat "foo" ["bar";"baz"]
+        Assert.AreEqual("barfoobaz", e7)
```

Comment:
@forki we ship products around the globe and have developed processes to protect us against inadvertently offending the sensibilities of our customers.  Foo + Bar is one of the more minor examples of things we look for.  Because it represents curse-words that some although not all may find offensive.